runtime/test-runtime/src/lib.rs

```diff
@@ -463,6 +463,7 @@ impl pallet_vesting::Config for Runtime {
 impl pallet_sudo::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type RuntimeCall = RuntimeCall;
+	type WeightInfo = ();
```

Comment:
we need to run the benches, and use an actual weight for of runtimes.

Reply:
Yeah, can we call a bot to do that like is done on the Substrate repo?

Reply:
yes, but I think it wont work until substrate is not merged
You can copy the weights from substrate. During the release process they will be replaced by actual weights.

Reply:
Thanks, will do that.

Reply:
Done for Rococo/Westend. For the testing runtimes it does not matter.